src/main/java/com/dreammaster/scripts/ScriptLogisticPipes.java

```diff
@@ -1262,6 +1256,114 @@ private void recipes1() {
                 .itemOutputs(getModItem(LogisticsPipes.ID, "item.PipeItemsBasicLogistics", 32, 0, missing))
                 .fluidInputs(FluidRegistry.getFluidStack("molten.solderingalloy", 72)).duration(40 * SECONDS)
                 .eut(TierEU.RECIPE_LV).addTo(assemblerRecipes);
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Iron, 4),
+                        getModItem(IndustrialCraft2.ID, "itemPartCircuit", 1, 0, missing),
+                        GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.Gold, 4L),
+                        GT_Utility.getIntegratedCircuit(18))
+                .itemOutputs(getModItem(LogisticsPipes.ID, "item.itemModule", 4, 0, missing))
+                .fluidInputs(FluidRegistry.getFluidStack("molten.lead", 144)).duration(30 * SECONDS)
+                .eut(TierEU.RECIPE_LV).addTo(assemblerRecipes);
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Iron, 4),
+                        ItemList.Circuit_Basic.get(1L),
+                        GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.Gold, 4L),
+                        GT_Utility.getIntegratedCircuit(18))
+                .itemOutputs(getModItem(LogisticsPipes.ID, "item.itemModule", 4, 0, missing))
+                .fluidInputs(FluidRegistry.getFluidStack("molten.lead", 144)).duration(30 * SECONDS)
+                .eut(TierEU.RECIPE_LV).addTo(assemblerRecipes);
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Iron, 4),
+                        ItemList.Circuit_Microprocessor.get(1L),
```

Comment:
is this on purpose? These itemlist circuits should only be used if you want to forbid unification (multiple occurences, not just here)